connecting to Carbon at 127.0.0.1:2003 (localhost)
zookeeper error on zoo_create, path /rtb-test/serviceClass/rtbAgentConfiguration/agentConfigurationService: node exists

--------------------------[Exception thrown]---------------------------
time:   2014-06-24T17:16:02
type:   ML::Exception
pid:    7567; tid: 7567
what:   Zookeeper error on zoo_create, path /rtb-test/serviceClass/rtbAgentConfiguration/agentConfigurationService: node exists
stack:
00: 0x0x7f566219adb7 at __cxa_throw + 0x27 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libexception_hook.so + 0xdb7/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libexception_hook.so(__cxa_throw+0x27) [0x7f566219adb7]
01: 0x0x7f566078e553 at Datacratic::ZookeeperConnection::checkRes(int, int&, char const*, char const*) + 0xe3 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x118553/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection8checkResEiRiPKcS3_+0xe3) [0x7f566078e553]
02: 0x0x7f566078e151 at Datacratic::ZookeeperConnection::createNode(std::string const&, std::string const&, bool, bool, bool, bool) + 0x441 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x118151/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection10createNodeERKSsS2_bbbb+0x441) [0x7f566078e151]
03: 0x0x7f5660745cd5 at Datacratic::ZookeeperConfigurationService::setUnique(std::string const&, Json::Value const&) + 0x285 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xcfcd5/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic29ZookeeperConfigurationService9setUniqueERKSsRKN4Json5ValueE+0x285) [0x7f5660745cd5]
04: 0x0x7f5660730d84 at Datacratic::ServiceBase::registerServiceProvider(std::string const&, std::vector<std::string, std::allocator<std::string> > const&) + 0x1c4 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xbad84/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic11ServiceBase23registerServiceProviderERKSsRKSt6vectorISsSaISsEE+0x1c4) [0x7f5660730d84]
05: 0x0x7f5661af8beb at RTBKIT::AgentConfigurationService::init() + 0x14b in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libagent_configuration.so + 0x91beb/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libagent_configuration.so(_ZN6RTBKIT25AgentConfigurationService4initEv+0x14b) [0x7f5661af8beb]
06: 0x0x40bc5b at main + 0x3fb in build/x86_64/bin/agent_configuration_service_runner + 0xbc5bbuild/x86_64/bin/agent_configuration_service_runner(main+0x3fb) [0x40bc5b]
07: 0x0x7f5660a1576d at __libc_start_main + 0xed in /lib/x86_64-linux-gnu/libc.so.6 + 0x2176d/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xed) [0x7f5660a1576d]
08: 0x0x40bf75 in build/x86_64/bin/agent_configuration_service_runner + 0xbf75build/x86_64/bin/agent_configuration_service_runner() [0x40bf75]

terminate called after throwing an instance of 'ML::Exception'
  what():  Zookeeper error on zoo_create, path /rtb-test/serviceClass/rtbAgentConfiguration/agentConfigurationService: node exists
